### DISCUSSION TITLE
Android TV / ADB - Touchpad

### DISCUSSION THREAD

**Discussion by reinuke** [PRProd/HA-Firemote#340] [Q&A] [answered]
Is there a way to configure a Touchpad? To get away from pressing the arrow keys.

**Comment by PRProd** [comment, marked as answer]
@reinuke ~

The short answer is no.  The long answer is maybe, but probably not soon.

To understand why, it's useful to know how Firemote sends commands to your media device.  Please read the section in the Firemote wiki: [How does Firemote Send Commands to my Devices?](https://github.com/PRProd/HA-Firemote/wiki#how-does-firemote-send-commands-to-my-devices)

Since there aren't any easy to use ways of sending touchpad style commands to a media device through Home Assistant, I have not explored this path yet.  My main goal for the project was to create a realistic looking Home Assistant card that would function just like my physical remote control.  Since my physical remote controls do not have any touchpads (with the exception of Apple TV), I haven't put any effort into looking at it.

That being said, I know that many Android based media devices (not Apple TV or Roku) will accept a touchpad style swipe command.  In fact, I think I have seen some HACS cards that do this already.  I would need to do some personal research first before I can say "yes" or "no" to the idea.

**Comment by reinuke** [comment]
I figured as much., but didn't see it discussed..
Would probably require some kind of an "integration" as well..
Thanks for the reply though 😇
